src/app/shared.module.ts

```diff
@@ -0,0 +1,71 @@
+import { NgModule } from '@angular/core';
+
+import { CommonModule } from '@angular/common';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCardModule } from '@angular/material/card';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
+import { MatInputModule } from '@angular/material/input';
+import { MatListModule } from '@angular/material/list';
+import { MatMenuModule } from '@angular/material/menu';
+import { MatProgressBarModule } from '@angular/material/progress-bar';
+import { MatProgressSpinnerModule } from '@angular/material/progress-spinner';
+import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatTooltipModule } from '@angular/material/tooltip';
+
+import { FlexLayoutModule } from '@angular/flex-layout';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { HttpClientModule } from '@angular/common/http';
+import { NgPipesModule } from 'ngx-pipes';
+
+import { FullscreenButtonComponent } from './components/fullscreen-button/fullscreen-button.component';
+import { HillBackgroundComponent } from './components/hill-background/hill-background.component';
+import { SentenceCasePipe } from './utils/string-utils';
+
+
+
+const ANGULAR_MATERIAL_MODULES = [
+  MatProgressSpinnerModule,
+  MatProgressBarModule,
+  MatButtonModule,
+  MatIconModule,
+  MatFormFieldModule,
+  MatInputModule,
+  MatCardModule,
+  MatMenuModule,
+  MatSnackBarModule,
+  MatDialogModule,
+  MatListModule,
+  MatTooltipModule,
+];
```

Comment:
Oh, you got rid of some of the material components we weren't using—nice catch! :)